pkg/indexmanagement/scripts.go

```diff
@@ -11,6 +11,7 @@ code=$(curl "$ES_SERVICE/$POLICY_MAPPING-write/_rollover?pretty" \
   --key /etc/indexmanagement/keys/admin-key \
   -HContent-Type:application/json \
   -XPOST \
+  -H"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
```

Comment:
if we're going to use a SA token, i dont think we need to provide the `--cert` and `--key` args

Reply:
doesn't the cert/key allow mutual tls at the connection level?  This was the only reason I added it.  not sure if its relevent

Reply:
right, providing the cert and key makes this a mTLS request which I don't think we need to connect to ES (it should be optional)

Reply:
@ewolinetz by optional are you suggesting we should allow user's to make it optional or do you believe in this case it is save to remove cert and key?

Reply:
in this case it is safe to remove